make/Makefile.molecule.mk

```diff
@@ -130,19 +126,11 @@ endif
 MOLECULE_WAIT_RETRIES_ARG ?= --env MOLECULE_WAIT_RETRIES=${MOLECULE_WAIT_RETRIES}
 
 .prepare-force-molecule-build:
-ifeq ($(DORP),docker)
```

Comment:
Can't I build using docker? This doesn't have anything to do with running molecule. This only affects builds, which may be done on a machine that does not have podman. 

I would keep DORP support for things like this.

Reply:
Sure you could build with docker and run with podman... but you'd still need to run with podman and therefore have podman installed on the same system you are building on so why not just build with podman? Part of the motivation for removing the `DORP` option was to make it more clear that you can't specify `DORP=docker` and expect the molecule tests to run with docker since they don't support docker anymore and need to be run with podman.

Reply:
not if I'm building on one machine and running on another.... I could build the image and push to the other machine.

Reply:
my motivation: I'm trying to figure out if this breaks my mazz basement setup  :)

Reply:
Yeah I guess you could build your dev/test image on one machine that only has docker, push it to quay, then pull it on another machine that has podman and run it there but does anyone do that?

> my motivation: I'm trying to figure out if this breaks my mazz basement setup :)

If mazz basement doesn't have both docker and podman installed then it won't ever be able to run the tests but I guess you could build them on mazz basement.

Reply:
> If mazz basement doesn't have both docker and podman

It does not. I do not have docker.